woocommerce/woocommerce-ios
# Lightweight storefront: Fix UI glitches

Closes: #11665 


## Description

This PR add fixes for the minor UI glitches:
- Explicitly set the theme settings view to display its navigation view in stack style. The issue was due to the view being presented from UIKit, which determines the style of the navigation view as split view based on the device's size class. The safest way to work with Navigation view is to set the desired style when you are in doubt, or switch to `NavigationStack` when we drop iOS 15.
- Added a bottom padding to the "No credit card required" on the free trial summary screen.

## Testing instructions

- Build the app to an iPad and log in to a WPCom store.
- Switch to Menu tab > Settings > Themes. Notice that the theme setting view is not displayed in a split view.
- Navigate back to the Menu tab and open the store picker.
- Select Add a Store > Create a new store. Notice that the "No credit card required" has a padding at the bottom.

Please feel free to confirm that the screens still look correctly on iPhone.

## Screenshots

Free trial summary | Theme settings
--- | ---
<img src="https://github.com/woocommerce/woocommerce-ios/assets/5533851/117d9bd8-c92a-41fa-a181-55956813433a" width=320 /> |  <img src="https://github.com/woocommerce/woocommerce-ios/assets/5533851/33d9db9a-5a81-42b6-b9a9-af357f59ce9d" width=320 />


---
